js/adapt-aboutUsItemView.js

```diff
@@ -5,108 +5,45 @@
-                'drawer:openedItemView': this.remove,
-                'drawer:triggerCustomView': this.remove
-            });
-        },
-
-        onAboutUsItemClicked: function(event) {
-            event && event.preventDefault();
-            Adapt.trigger('aboutUs:descriptionOpen', this.model.cid);
-        },
-
-        toggleAboutUsItemDescription: function() {
-            if (this.model.get('_isDescriptionOpen')) {
-                this.hideAboutUsItemDescription();
-            } else {
-                this.showAboutUsItemDescription();
-            }
-        },
-
-        showAboutUsItemDescription: function() {
-            var $aboutUsItemTopic = this.$('.js-aboutus-item-topic-click');
-            var description = $aboutUsItemTopic.addClass('is-selected')
-                .siblings('.aboutus__item-body')
-                .slideDown(200, function() {
-                    Adapt.a11y.focusFirst(this.$el, {defer: true});
-                }.this);
-            $aboutUsItemTopic.attr('aria-expanded', true);
-            this.model.set('_isDescriptionOpen', true);
-        },
-
-        hideAboutUsItemDescription: function() {
-            this.$('.aboutus__item-body').stop(true, true)
-                .slideUp(200);
-            this.model.set('_isDescriptionOpen', false);
-
-            this.$('.js-aboutus-item-topic-click').removeClass('is-selected')
-                .attr('aria-expanded', false);
-        },
-
-        descriptionOpen: function(viewId) {
-            if (viewId === this.model.cid) {
-                this.toggleAboutUsItemDescription();
-            } else if (this.model.get('_isDescriptionOpen')) {
-                this.hideAboutUsItemDescription();
-            }
-        },
-
-        onAboutUsItemVisibilityChange: function() {
-            if (this.model.get('_isDescriptionOpen')) {
-                this.hideAboutUsItemDescription();
-            }
-            if (this.model.get('_isVisible')) {
-                this.$el.removeClass('u-display-none');
-            } else {
-                this.$el.addClass('u-display-none');
-            }
-        }
-
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { templates } from 'core/js/reactHelpers';
+
+class AboutUsItemView extends Backbone.View{
+  initialize() {
+    this.listenTo(Adapt, {
+        'remove drawer:closed': this.remove,
+        'aboutUs:descriptionOpen': this.descriptionOpen
+    });
+    this.setupModel();
+    this.listenTo(this.model, 'change:_isVisible', this.onAboutUsItemVisibilityChange);
+    this.render();
+  }
+
+  setupModel() {
+    this.model.set({
+        '_isVisible': true,
+        '_isDescriptionOpen': false
+    });
+  }
+  
+  render() {
+    ReactDOM.render(<templates.aboutUsItem {...data} />, this.el);
+
+    _.defer(_.bind(function() {
+      this.postRender();
+    }, this));
+
+    return this;
+  }
+
+  postRender() {
+    this.listenTo(Adapt, {
```

Comment:
Minor: It seems like this behaviour could be moved to the initialize with the other listeners. I'm not sure why somebody had it down here in the first place, but would remove the need for a postRender function altogether